src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -204,11 +206,14 @@ export class SamCliLocalInvokeInvocation {
         this.dockerNetwork = params.dockerNetwork
         this.skipPullImage = skipPullImage
         this.debuggerPath = params.debuggerPath
+        // Enterprise!
+        this.invokerContext = new DefaultSamCliProcessInvokerContext()
     }
 
     public async execute(timeout?: Timeout): Promise<void> {
         await this.validate()
 
+        const samCommand = this.invokerContext.cliConfig.getSamCliLocation() ?? 'sam'
```

Comment:
This will be a full path, right? Is there any risk that any of the other values are relative and won't path correctly with a full SAM path being executed?

Reply:
Process image path does not affect CWD nor its arguments in any way. Can you give an example?

Somewhat related question is, why this module doesn't use `SamCliProcessInvoker` like all other samCliX modules. But this fix is least-risk for now.